sriov/common/test_utils_mock.py

```diff
@@ -1,5 +1,7 @@
 import unittest
 from mock import Mock, patch
+import sys, os
+sys.path.insert(1, os.path.dirname(os.path.abspath(__file__)))
```

Comment:
@spenced any suggestion on this line in order to this script from a different folder?